translations/README.spanish.md

```diff
@@ -42,38 +43,43 @@ Leer más en [hasura.io](https://hasura.io) y en [docs](https://hasura.io/docs).
 <!-- markdown-toc start - Don't edit this section. Run M-x markdown-toc-refresh-toc -->
 **Tabla de contenidos**
 
-- [Inicio rápido:](#inicio-rápido)
+- [Motor Hasura GraphQL](#motor-hasura-graphql)
+  - [Características](#características)
+  - [Tabla de contenidos](#tabla-de-contenidos)
+  - [Inicio rápido:](#inicio-rápido)
     - [Despliegue en un click con Heroku](#despliegue-en-un-click-con-heroku)
+    - [Otras opciones de despliegue en un clic](#otras-opciones-de-despliegue-en-un-clic)
     - [Otros métodos de despliegue](#otros-métodos-de-despliegue)
-- [Arquitectura](#arquitectura)
-- [Herramientas client-side](#herramientas-client-side)
-- [Agregar lógica de negocio](#agregar-lógica-de-negocio)
+  - [Arquitectura](#arquitectura)
+  - [Herramientas client-side](#herramientas-client-side)
+  - [Agregar lógica de negocio](#agregar-lógica-de-negocio)
     - [Esquemas remotos](#esquemas-remotos)
     - [Ejecutar webhooks con eventos de la base de datos](#ejecutar-webhooks-con-eventos-de-la-base-de-datos)
-- [Demos](#demos)
+    - [Datos derivados o transformación de datos](#datos-derivados-o-transformación-de-datos)
+  - [Demos](#demos)
     - [Aplicaciones en tiempo real](#aplicaciones-en-tiempo-real)
     - [Videos](#videos)
-- [Soporte y solución de problemas](#soporte-y-solución-de-problemas)
-- [Contribuir](#contribuir)
-- [Archivos de marca](#archivos-de-marca)
-- [Licencia](#licencia)
-- [Traducciones](#traducciones)
+  - [Soporte y solución de problemas](#soporte-y-solución-de-problemas)
+  - [Contribuir](#contribuir)
+  - [Archivos de la marca](#archivos-de-la-marca)
+  - [Licencia](#licencia)
+  - [Traducciones](#traducciones)
 
 <!-- markdown-toc end -->
 
 ## Inicio rápido:
 
 ### Despliegue en un click con Heroku
 
-La manera más rápida de probar Hasura es usando Heroku.
+La manera más rápida y sencilla de probar Hasura es a través de [Hasura Cloud](https://hasura.io/docs/cloud/1.0/manual/getting-started/index.html).
 
-1. Da click en el siguiente botón para desplegar el motor GraphQL en Heroku con el add-on gratuito de Postgres:
+1. Haga click en el siguiente botón para desplegar el motor GraphQL en Hasura Cloud, incluyendo el complemento de Postgres o utilizando una base de datos de Postgres existente:
 
-    [![Deploy to Heroku](https://www.herokucdn.com/deploy/button.svg)](https://heroku.com/deploy?template=https://github.com/hasura/graphql-engine-heroku)
+    [![Deploy to Hasura Cloud](https://graphql-engine-cdn.hasura.io/img/deploy_to_hasura.png)](https://cloud.hasura.io/)
 
 2. Abre la consola de Hasura
 
-   Visita `https://<app-name>.herokuapp.com` (*reemplaza \<app-name\> con el nombre de tu aplicacion*) para abrir la consola de administrador.
+    Haga clic en el botón "Iniciar consola" para abrir la consola Hasura.
```

Comment:
Maybe it should be "Initialize console" because it will be that way in the hasura console, IIUC.